src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import cromwell.engine.WorkflowManagerActor
+
+trait WorkflowManagerSystem {
+  val systemName = "cromwell-system"
+  val actorSystem = ActorSystem(systemName)
+
+  actorSystem.registerOnTermination({
+    actorSystem.log.info(s"$systemName shutting down")
+  })
```

Comment:
scanning around teh googelz it looks like a pretty even split between `registerOnTermination({asdasdf})`, `registerOnTermination(asdfasdf)` and `registerOnTermination {asdf}`. I'm going with the latter.